mdb_baxter_detection/ObjDet[] objects_detection
